pkg/workflow/controller/handlers/workflowtrigger/cron_job.go

```diff
@@ -74,14 +74,11 @@ func (m *CronTriggerManager) DeleteTrigger(wftKey string) {
 		wft.Cron.Stop()
 		wft.IsRunning = false
 		delete(m.CronTriggerMap, wftKey)
-	} else {
-		log.Warnf("failed to delete cronTrigger(%s), not exist\n", wftKey)
```

Comment:
Why remove this warning?

Reply:
The `Reconcile` method will delete and create the cron task even when processing creation event, and the deletion triggered by a creation event will always result in a `not exist` error. I think it is not much graceful.